mne/decoding/receptive_field.py

```diff
@@ -47,13 +48,18 @@ class ReceptiveField(BaseEstimator):
         Defines how predictions will be scored. Currently must be one of
         'r2' (coefficient of determination) or 'corrcoef' (the correlation
         coefficient).
+    patterns : bool
+        If True, inverse coefficients will be computed upon fitting according
+        to the procedure described by Haufe et al. [5]_. Defaults to False.
```

Comment:
can we give more information than this? E.g., the coefficients will be inverted using the covariance matrix of the inputs, and the cross-covariance of the inputs/outputs, according to [5]_...or something like that